types/cesium/index.d.ts

```diff
@@ -2873,7 +2878,7 @@ declare namespace Cesium {
         distanceDisplayCondition: Property;
         show: Property;
         material: MaterialProperty | Color;
-        hierarchy: Property;
+        hierarchy: Property | Cartesian3[] | PolygonHierarchy;
```

Comment:
This breaks our application with the following errors:
`error TS2339: Property 'getValue' does not exist on type 'Property | Cartesian3[] | PolygonHierarchy'.`

Reply:
I have pushed a fix here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44648

Reply:
you can remove the `getValue` from your code

Reply:
No, that will not work properly. Cesium takes any value set for `hierarchy` and automatically wraps it in `Property`. If you open this sandcastle and look at the log, you will see that `PolygonGraphics.hierarchy.positions` is `undefined`, even though it was originally set with a `PolygonHierarchy` and not a `Property`:

https://sandcastle.cesium.com/#c=lVLBbpxADP0VixMrbYdNm0oVIaummyg9VG3URrmEHibghVGHMfIMpKTKv3dYYIM2q0rlgObZ79mWn1vJ0Cp8RIZzMPgIG7SqqcTdLhamQbbDGzJOKoOcBouz1KSm9bqCEc0N6a4g49VDGYHGKafQCpnn4Z/UABhZYQxpcN3zAX87bnLMoR6UabDsSSOKYScBKBWy5Kzs4mmkjWTnX9K8E1um6hL7/vaCWXbh/SACeHOy+iBWywmevp0Bn1v9I3c6z70Qfy7GxzT4Z1RF6WJ4v+q/Pa+SDllJ/TIvaWJx/f3q6utIyTRZvKU6hq3UFufRT+QcVfPEs/8/D8vOyFjSKDQV3pFLxZg5kJm3xsKW2O/O+pV7UjzYM+fPTRLjksV+uWIvPdbpYmjhSqamKOGGqUZ2nSjQ3Und4P+3m5Th4rDzeD1PRNUthQe3tDiDYBkk1nUa15NHH1VVEztoWIdCRA6rWnsPbPTQZL/QiczuSvfUJJpLk1y1oPLzI8ftzZDW+sy20fqHesI0WCeR57+SapK5MsW3FlnLrqeVJ+svQ1AIkUQeHlc6Iv0g+aDyXw

Reply:
@jrafidi thanks. what a mess. I saw your fix, it still removes actual possible acceptable values by cesium. e.g : `semiMajorAxis: Property | number;` -> `semiMajorAxis: Property;`

Though, thanks for the support.

Reply:
Yes, I wish that we could describe different set/get types in typescript, but at the moment we can't.